server/events/vcs/gitlab_client.go

```diff
@@ -295,3 +295,7 @@ func (g *GitlabClient) DownloadRepoConfigFile(pull models.PullRequest) (bool, []
 func (g *GitlabClient) SupportsSingleFileDownload(repo models.Repo) bool {
 	return true
 }
+
+func (g *GitlabClient) PullIsLocked(baseRepo models.Repo, pull models.PullRequest) (bool, error) {
+	return false, fmt.Errorf("not implemented")
+}
```

Comment:
We shouldn't have to do this, would rather not touch implementors we are never going to use since this won't be for upstream. Best to use github client as is for new methods we want to support. See my suggestion above.

Reply:
Done!